### DISCUSSION TITLE
Welcome to PKNCA Discussions!

### DISCUSSION THREAD

**Discussion by billdenney** [humanpred/pknca#211] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smouksassi** [comment]
Hi Bill, Nice initiative. My name is Samer Mouksassi I am a Pharmacometrician working in Drug Development. I use R daily !
Great to have an R-based solution for NCA, especially for a simulation-reestimation methodology project where we want to compute NCA on thousands of simulated PK profiles.

**Reply by billdenney** [reply]
Hi Samer,

I'm glad that you find PKNCA useful.  And, thanks for sharing your usa case.  I also like it for calculating all the profiles with simulations -- it fits easily into that workflow.
